scripts/install_prerequisites.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
+
+
+sudo apt-get update
+ 
+sudo apt-get install git 
+
+sudo apt-get install unzip
+ sudo apt-get install default-jre
+
+sudo apt-get install curl python-setuptools python-dev python-pip 
+sudo pip install --upgrade pip
```

Comment:
[--upgrade](http://python-packaging-user-guide.readthedocs.org/en/latest/installing/#upgrading-packages) is the flag, pip is the package to upgrade. It makes sure that pip is upgraded to the latest version.